runners/wikiConsole/templates/buildNg.asciidoc

```diff
@@ -0,0 +1,19 @@
+buildNg command
+=============
+
+Prerequisites
+~~~~~~~~~~~~~~
+* You need to have either installed the devonfwIDE with an angular installation or installed https://angular.io/[Angular] seperatly on your system.
+* You need to have created or downloaded a Angular Project to a local directory.
+
+For this tutorial move to the angular directory by executing `cd <%= angularPath; %>`
+
+Execution
+~~~~~~
+<% if(outputPath){ %>
+If you want to compile the app to a custom output directory, you can execute the command `ng build --output-path (<%= outputPath; %>). The output path must me relative to your current workspace.
```

Comment:
You missed the ` after the command. And <%= outputPath; %> has not to be in brackets I think